draft-ietf-quic-recovery.md

```diff
@@ -848,9 +850,10 @@ and not fully utilize the congestion window due to this delay. A sender
 should not consider itself application limited if it would have fully
 utilized the congestion window without pacing delay.
 
-Bursting more than an intial window's worth of data into the network might
-cause short-term congestion and losses. Implemementations SHOULD either use
-pacing or reduce their congestion window to limit such bursts.
+Sending multiple packets into the network without any delay between them
+creates a burst of load that might cause short-term congestion and losses.
```

Comment:
```suggestion
creates a packet burst that might cause short-term congestion and losses.
```

Reply:
I don't think this description of burst is necessary though. It doesn't help with #2686, if that's what you were trying to address. I would drop this, but I realized that the existing statement is potentially misleading in what it says. (more discussion on this in #2686)

Reply:
I added the definition of burst, because I realize it might not be obvious?

I think the core of #2686 is that the existing text is unclear.  The existing text says you SHOULD not burst more than an initial window into the network(ever), with nothing specific to coming out of quiescence.

Reply:
>  The existing text says you SHOULD not burst more than an initial window into the network(ever), with nothing specific to coming out of quiescence.

Yes, I think this is the problem. I believe the intent here was to say that a sender SHOULD NOT do more than IW as an upper limit on bursts without recommending that as the correct limit for any bursting.

We can either specify that _all_ bursts should be limited to IW, irrespective of how much data is in the network already, or that bursts coming out of quiescence should be limited to IW. While I'm not sure how to change this text here, I'm trying to steer clear of saying something overly broad. (I'll suggest some text).

Reply:
If not out of quiescence, an IW burst would probably be bad. You could just wait a millisecond or two and repeat in badly designed pacing logic.

Reply:
I'd rather have it apply always, because I don't want to have one packet sent, and then a sender can burst the full window.  Making it specific to quiescence adds unnecessary complexity to the rules and may not accomplish the goal of mitigating the congestion caused to other flows of bursty sending.

I thought my existing text does that, but people are still confused, so better text would be greatly appreciated.

Reply:
The problem is that it is not clear when the next burst can be sent.